hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -205,6 +206,7 @@ public CacheConfig(Configuration conf, ColumnFamilyDescriptor family, BlockCache
     this.blockCache = blockCache;
     this.byteBuffAllocator = byteBuffAllocator;
     LOG.info("Created cacheConfig: " + this + (family == null ? "" : " for family " + family) +
+        (hri == null ? "" : " in region " + hri.getRegionNameAsString()) +
         " with blockCache=" + blockCache);
```

Comment:
Here this blockCache will just give object identifier as no toString() there. BlockCache will be single object within RS. So IMO there is no need to log that with every Store create.  Anyway if you do other comment, we will avoid it as CacheConfig#toString() wont be having BlockCache object

Reply:
Yes, maybe we can log CacheConfig in the BlockCacheFactory#createBlockCache and remove it from here